docs/Developers/Review.rst

```diff
@@ -140,6 +140,11 @@ For the Reviewers
    Bears require special attention during testing.
    `Testing Bears <http://api.coala.io/en/latest/Developers/Testing_Bears.html>`__
    provides a guideline for how to test bears.
+-  Check if the changes in the PR have been pushed from the master
+   branch. If this has happened, then drop a comment telling the
+   code-writer that it is not a good practice to do, but also indicate
```

Comment:
Remove `to do`

Reply:
How does `contributor` sound? `code-writer` doesn't look that good.

Reply:
In the third point under `For the reviewers` in https://api.coala.io/en/latest/Developers/Review.html, the assignee is referred to as the code-writer. plz take a look into that. I made my changes according to it.

Reply:
Yes, I saw that already. On the same page the word contributor has been used and that looks better.

Reply:
Okay, but contributor would still look better. Besides, I just read the page, and the word contributor has been used there too.